packages/react/src/generators/application/schema.json

```diff
@@ -161,7 +161,7 @@
     "compiler": {
       "type": "string",
       "description": "The compiler to use.",
-      "enum": ["babel", "swc"],
+      "enum": ["swc", "babel"],
       "default": "babel"
```

Comment:
same here

Reply:
The React SWC setup wasn't 100% correct with CSS-in-JS, but that's been fixed here https://github.com/nrwl/nx/pull/17295

We can make it default now.